### PR TITLE
Translation of Training documentation in french

### PR DESCRIPTION
**Pull request by justicengom**
docs/basic_training/tower.fr.md

### CODE REVIEW

**Comment by abhi18av** [comment]
Pinging @maxulysse for a review here please :)

File: docs/basic_training/tower.fr.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+Les principales caractéristiques de Nextflow tower sont les suivantes
+
+-   Le lancement de pipelines préconfigurés en toute simplicité.
+-   L'intégration programmatique pour répondre aux besoins d'une organisation.
+-   La publication de pipelines dans des espaces de travail partagés.
+-   La gestion de l'infrastructure nécessaire à l'exécution d'analyses de données à grande échelle.
+
+!!! conseil
+
+    [Sign up](https://cloud.tower.nf/) pour essayer Tower gratuitement ou demander une [demo](https://seqera.io/demo/) pour des déploiements dans votre propre environnement sur site ou en could.
```
Comment (maxulysse) [inline docs/basic_training/tower.fr.md:21]:
```suggestion
    [Inscrivez-vous](https://cloud.tower.nf/) pour essayer Tower gratuitement ou demander une [demo](https://seqera.io/demo/) pour des déploiements dans votre propre environnement sur site ou en could.
```

File: docs/basic_training/tower.fr.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
+---
+titre: Nextflow Tower
+description: Demarrer avec Nextflow Tower
+---
+
+# Demarrer avec Nextflow Tower
+
+## Concept de Base
+
+Nextflow Tower est le poste de commande centralisé pour la gestion des données et des workflows. Il apporte la surveillance, la journalisation et l'observabilité aux flux de travail distribués et simplifie le déploiement des flux de travail sur n'importe quel nuage, cluster ou ordinateur portable. Dans la terminologie de Tower, un flux de travail est ce sur quoi nous avons travaillé jusqu'à présent, et les pipelines sont des workflows préconfigurés qui peuvent être utilisés par tous les utilisateurs d'un espace de travail. Il est composé d'un rrepositoire de workflows, de paramètres de lancement et d'un environnement de calcul. Nous nous en tiendrons à ces définitions dans cette section.
```
Comment (maxulysse) [inline docs/basic_training/tower.fr.md:10]:
```suggestion
Nextflow Tower est le poste de commande centralisé pour la gestion des données et des workflows. Il apporte la surveillance, la journalisation et l'observabilité aux flux de travail distribués et simplifie le déploiement des flux de travail sur n'importe quel cloud, cluster ou ordinateur portable. Dans la terminologie de Tower, un flux de travail est ce sur quoi nous avons travaillé jusqu'à présent, et les pipelines sont des workflows préconfigurés qui peuvent être utilisés par tous les utilisateurs d'un espace de travail. Il est composé d'un rrepositoire de workflows, de paramètres de lancement et d'un environnement de calcul. Nous nous en tiendrons à ces définitions dans cette section.
```

File: docs/basic_training/tower.fr.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+Vous pouvez utiliser Tower via l'option `-with-tower` lors de l'utilisation de la commande `nextflow run`, via l'interface graphique **en ligne** ou via l'interface utilisateur **API**.
+
+### Via la commande `nextflow run`.
+
+Créez un compte et connectez-vous à Tower.
+
+**1. Créer un nouveau jeton**
+
+Vous pouvez accéder à vos tokens à partir du menu déroulant **Réglages** :
```
Comment (maxulysse) [inline docs/basic_training/tower.fr.md:33]:
```suggestion
Vous pouvez accéder à vos jetons à partir du menu déroulant **Réglages** :
```

File: docs/basic_training/tower.fr.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+
+Pour soumettre un pipeline à un [espace de travail](https://help.tower.nf/getting-started/workspace/) en utilisant l'outil de ligne de commande Nextflow, ajoutez l'ID de l'espace de travail à votre environnement. Par exemple :
+
+```bash
+export TOWER_WORKSPACE_ID=000000000000000
+```
+
+L'identifiant de l'espace de travail se trouve sur la page de présentation des espaces de travail de l'organisation.
+
+**5. Exécuter Nextflow avec tower**
```
Comment (maxulysse) [inline docs/basic_training/tower.fr.md:69]:
```suggestion
**5. Exécuter Nextflow avec Tower**
```

**Comment by mribeirodantas** [comment]
Great! Thanks, @justicengom and @maxulysse 🥳

**Comment by abhi18av** [comment]
Good speed @justicengom ! 🚀

**Pull request merged**